gradle.properties

```diff
@@ -21,3 +20,5 @@ githubBranch=master
 githubCoreBranch=3.2.x
```

Comment:
set  `githubCoreBranch` to `3.5.x`

Reply:
Done! Also updated the version to 3.2.0-SNAPSHOT